cmd/eval.go

```diff
@@ -219,10 +236,16 @@ func eval(args []string, params evalCommandParams) (err error) {
 	}
 
 	var p *profiler.Profiler
-
-	if params.profile {
+	if isProfilingEnabled(params) {
 		p = profiler.New()
 		regoArgs = append(regoArgs, rego.Tracer(p))
+
+		// include metrics as part of the profiler result
+		if m == nil {
```

Comment:
Refactor the command line argument handling to set params.metrics=true if params.profile=true. We don't have to duplicate the logic here.

Reply:
Done